.github/workflows/coverage-linux.yml

```diff
@@ -0,0 +1,35 @@
+name: coverage-linux
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+env:
+  PYTHON_VERSION: 3.9
+  FLAKY_TESTS: dontcare
+
+jobs:
+  coverage-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ env.PYTHON_VERSION }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - name: Environment Information
+        run: npx envinfo
+      - name: Build
+        run: make build-ci -j2 V=1 CONFIG_FLAGS="--error-on-warn"
+      - name: Test
+        run: NODE_V8_COVERAGE=coverage/tmp make run-ci -j2 V=1 TEST_CI_ARGS="-p actions" || exit 0
```

Comment:
Are we expecting tests to fail with coverage enabled? If so we should change to 
```suggestion
        run: NODE_V8_COVERAGE=coverage/tmp make run-ci -j2 V=1 TEST_CI_ARGS="-p dots" || exit 0
```
otherwise we get "Unchanged files with check annotations" like the following in the "Files changed" tab on pull requests for each failing test (up to a maximum of 10):
![image](https://user-images.githubusercontent.com/5445507/96107166-ff5b6480-0ed3-11eb-86a6-15ae8a6707ee.png)

Reply:
It's expected that a few tests will currently fail, due to the modifications coverages makes to the inspector session.

Along with following your suggestion, I will leave a TODO comment to fix this (_good chance for someone to make a contribution_).